package.json

```diff
@@ -5,19 +5,20 @@
   "main": "index.js",
   "scripts": {
     "pretest": "npm install",
-    "test": "eslint lib/**/*.js test/**/*.js index.js && mocha --exit --require should --reporter spec --check-leaks"
+    "test": "eslint lib/**/*.js test/**/*.js index.js && mocha --exit --require should --reporter spec --check-leaks",
+    "test-http2": "HTTP2_TEST=1 mocha --exit --require should --reporter spec --check-leaks"
   },
   "dependencies": {
     "methods": "~1.1.2",
-    "superagent": "3.8.2"
+    "superagent": "git+https://github.com/visionmedia/superagent.git"
   },
   "devDependencies": {
     "body-parser": "^1.18.2",
     "cookie-parser": "^1.4.3",
     "eslint": "^4.19.1",
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "2.11.0",
-    "express": "^4.16.3",
+    "express": "git+https://github.com/sogaani/express.git#initial-support-http2",
```

Comment:
Similar to https://github.com/visionmedia/superagent/pull/1414#issuecomment-421978652, it would be good to depend directly on `express@^4.18.1`.

PS: `express@^4.18.1` is in the [master branch package.json](https://github.com/visionmedia/supertest/blob/5543d674cf9aa4547927ba6010d31d9474950dec/package.json#L45) at the time of writing.